driver/call.py

```diff
@@ -50,6 +50,11 @@ def fail(exception, exitcode):
     else:
         return set_limits
 
+def _get_env(prepend_python_path=None):
```

Comment:
This could use a doc string or comment to explain why we need it. In general adding something to the (Python)path feels a bit hacky. As far as I understand, we only do this because the translator is not installed in the way that other Python packages are. In the long run, we might want to change that and publish the translator as its own package. Then, ti would be installed as a dependency and messing with the path would not be necessary. I suggest to mention this here in a comment.

Reply:
I added a doc string.